packages/SwingSet/src/kernel/state/vatKeeper.js

```diff
@@ -22,6 +22,13 @@ export function makeVatKeeper(
 ) {
   insistVatID(vatID);
 
+  const enableKDebug = false;
+  function kdebug(...args) {
```

Comment:
kdebug is defined twice, once here and once in kernelKeeper.js.  Is this intentional or a mistake?

Reply:
It was intentional, in that I thought we'd want separate control of the kdebug flags in different files, but Brian's suggestion was to merge them into a single version with a single flag. I'll do that separately.